### PR TITLE
Add 3D model type to constants

### PR DESCRIPTION
**Pull request by foroveralls**
## Fixes

Fixes #[#658] by @[AetherUnbound]

## Description

This PR updates the file media_types.py by adding the media type "3D Model" to the list of possible media types to allow the ingestion server to perform data refreshes for that media type. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: ingestion_server/ingestion_server/constants/media_types.py [hunk 1]
```diff
@@ -4,5 +4,6 @@
 
 AUDIO_TYPE = "audio"
 IMAGE_TYPE = "image"
+3D_MODEL_TYPE = "model_3d"
```
Comment (dhruvkb) [inline ingestion_server/ingestion_server/constants/media_types.py:7]: Python identifiers cannot start with a number. Please change this.
```suggestion
MODEL_3D_TYPE = "model_3d"
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: ingestion_server/ingestion_server/constants/media_types.py (hunk 1 above)
Comment (foroveralls) [reply ingestion_server/ingestion_server/constants/media_types.py:7]: whoops of course, thank you

**Review by AetherUnbound — APPROVED** [review]
We have one action failing due to an unrelated issue - this looks good, thanks for the contribution @foroveralls!

**Comment by sarayourfriend** [comment]
Will merging this now allow ingestion to work before there are 3d models to ingest?

**Comment by dhruvkb** [comment]
@sarayourfriend yes, adding this should not impact ingestion. The passing CI gives me confidence that everything that works now remains functional after this change.

**Pull request merged**